docs/wiki/EL/JSON-RPC.md

```diff
@@ -0,0 +1,113 @@
+# JSON-RPC
+
+---
+
+The JSON-RPC specification is a remote procedure call protocol encoded in JSON based on [OpenRPC](https://open-rpc.org/getting-started). It allows calling functions on a remote server, and for the return of the results.
+It is part of the Execution API specification which provides a set of methods to interact with the Ethereum blockchain.
+It is better known to be the way of how the clients interact with the network, even how the consensus layer (CL) and the execution layer (EL) interact.
```

Comment:
> how the clients interact with the network

This can be misleading, I would rephrase to something like 'users interact with the network using a client'..

Reply:
Also when mentioning it's used for EL<>CL communication, I would explicitly said that engine API uses it as well but it's another authenticated interface not to be confused with user facing